packages/core/src/modules/oob/messages/OutOfBandInvitation.ts

```diff
@@ -99,13 +99,21 @@ export class OutOfBandInvitation extends AgentMessage {
 
   // TODO: this only takes into account inline didcomm services, won't work for public dids
   public getRecipientKeys(): Key[] {
-    return this.services
-      .filter((s): s is OutOfBandDidCommService => typeof s !== 'string' && !(s instanceof String))
+    return this.getServices()
+      .filter((s): s is OutOfBandDidCommService => typeof s !== 'string')
       .map((s) => s.recipientKeys)
       .reduce((acc, curr) => [...acc, ...curr], [])
       .map((didKey) => DidKey.fromDid(didKey).key)
   }
 
+  // shorthand for services without the need to deal with the String DIDs
```

Comment:
Great addition! I was struggling with the class transformer way too much and settled for String instances...